frontend/src/features/permits/components/dashboard/tests/integration/fixtures/getVehicleInfo.ts

```diff
@@ -130,7 +131,7 @@ export const getDefaultTrailerSubTypes = () => [
     type: "Trailer Type C",
     description: "Trailer Type C.",
   },
-  { ...TROS_INELIGIBLE_TRAILERS[0] },
+  { ...getIneligibleTrailerSubtypes(DEFAULT_PERMIT_TYPE)[0] },
```

Comment:
Is there backend validation for ineligible types? Or is it going to part of the rules engine?

Reply:
No, none at the moment. It's completely done by the frontend at the moment.